src/DiIiS-NA/D3-GameServer/GSSystem/AISystem/Brains/HirelingBrain.cs

```diff
@@ -151,17 +151,10 @@ public override void Think(int tickCounter)
 		protected virtual int PickPowerToUse()
 		{
 			// randomly used an implemented power
```

Comment:
I meant this comment above. So according to this comment we should try to pick random implemented power instead of just random